lib/vquic/ngtcp2.c

```diff
@@ -1660,6 +1660,12 @@ static ssize_t ngh3_stream_send(struct Curl_easy *data,
     return -1;
   }
 
+  // Reset post upload buffer after resumed.
+  if(data->set.postfields) {
```

Comment:
This seems too specific for POST. Shouldn't this rather check if `stream->upload_mem` is set?